Automattic/wp-calypso
# Reading Settings: Add "Syndication feeds" setting

#### Proposed Changes

* Add "Syndication feeds" setting to the new Reading Settings page

#### Testing Instructions



* Run local version of calypso
* Go to http://calypso.localhost:3000/settings/reading/<YOUR-BLOG>
* Ensure the setting:
  * displays the correct value
  * can be modified and saved
* Validate that the modified value actually affects how many posts are displayed on a blog posts page
* Smile

#### Pre-merge Checklist





Closes #70413 

![Screen Shot 2022-12-08 at 14 51 46](https://user-images.githubusercontent.com/2019970/206464329-a19fd67c-cf0f-41e7-834a-5d3d39c5f18e.png)
